.github/workflows/zap-owasp.yaml

```diff
@@ -38,3 +40,15 @@ jobs:
           cmd_options: "-a -d -T 5 -m 2"
           issue_title: OWASP Baseline - Backend
           fail_action: false
```

Comment:
would this get triggered if the failure is the secondary rate limit? We could pass the "retry-on-failure" directly here

Reply:
Oh by creating a `retry-on-failure` action? Might be a good idea anyways to reuse if there are other flaky workflows in the future.